src/d3d12/D3D12.h

```diff
@@ -77,4 +78,7 @@ struct D3D12
     Paths& m_paths;
     Window& m_window;
     Options& m_options;
+    
+    std::atomic_bool m_delayedTrapInput{ false };
```

Comment:
Those should be wrapped in setters/getters

Reply:
I think I dont understand you now?
These are set from DelayedSetTrapInputImGui function which was added.

I dont think there should be ever need for gets for this. Setter should be in place.

Reply:
Can change it but like I dont understand now what would you like changed exactly :)

Reply:
Ah right it's because you access them from a static